packages/EasyErrorHandler/src/Exceptions/BadRequestException.php

```diff
@@ -6,13 +6,7 @@
 
 abstract class BadRequestException extends BaseException
 {
-    /**
-     * @var int
-     */
-    protected $statusCode = 400;
+    protected int $statusCode = 400;
 
-    /**
-     * @var string
-     */
-    protected $userMessage = 'exceptions.bad_request';
+    protected ?string $userMessage = 'exceptions.bad_request';
```

Comment:
Do we need that to be nullable? Here and in other places.

Reply:
I'm not sure. It is nullable here because of the `\EonX\EasyErrorHandler\Interfaces\Exceptions\TranslatableExceptionInterface` interface.

Reply:
And we set the default value here `\EonX\EasyErrorHandler\Builders\UserMessageErrorResponseBuilder::doBuildValue` if it is `null`.

Reply:
ok